integration-tests/tests/mocked/get-question/step-functions-constants.ts

```diff
@@ -0,0 +1,22 @@
+import fs from "fs";
+import path from "path";
+
+const EVENT_FILE = path.join(__dirname, "../../../events/sfn_valid_input.json");
+const STATE_MACHINE_FILE = path.join(
+  __dirname,
+  "../../../../step-functions/get-question.asl.json"
```

Comment:
Check this path, might need to go up 1 more level

Reply:
No just checked it is correct